.travis.yml

```diff
@@ -6,6 +6,10 @@ sudo: false
 
 bundler_args: --without development
 
+branches:
+  only:
+    - master
+
```

Comment:
Could you please be a little more educational in your reviews? I can't learn anything from "don't do this"... :disappointed:

Anyways, it's now removed, the double CI run is now back there.

![doublerun](https://cloud.githubusercontent.com/assets/2887858/23104513/1ef9d304-f6ae-11e6-95a1-4a05aebf1e8c.png)